docs/build-module-cr.md

```diff
@@ -0,0 +1,34 @@
+| [Home](../README.md) |
+|----------------------|
+
+## CR for Building a new Module
+
+This task entails following sub-tasks:
+
+1. Application administrator creates a new CR for building a new module. For creating a new CR refer to the [New Change Request from Production Environment](#new-change-request-from-production-environment).
+
+2. Content Developer creates a new module. For creating a new module, refer to the article [Creating a New Module](https://docs.fortinet.com/document/fortisoar/7.3.1/administration-guide/97786/application-editor#Creating_a_New_Module).
```

Comment:
Content Developer creates a new module in development environment.

Reply:
Fixed